help/C/Help_ch_Transactions.xml

```diff
@@ -903,13 +903,20 @@ Author:
         <listitem><para><emphasis role="bold"><link linkend="trans-import-MT940-MT942-DTAUS">DTAUS</link></emphasis>
              (.DTAUS)  Banking data Exchange File;</para></listitem>
       </itemizedlist>
+      <note><para>Files using these formats do not necessarily use the file extensions shown above.</para></note>
       <para>These import methods can be accessed from the <guisubmenu>Import</guisubmenu> submenu
             of the main menu <guimenu>File</guimenu> entry, along with methods for importing
```

Comment:
I would prefer
`These import methods can be accessed from the <menuchoice><guimenu>File</guimenu><guisubmenu>Import</guisubmenu></menuchoice> submenu, ...`

Reply:
Also done . Bob I'm a bit unsure of the procedure here once a PR has been created. If I just push these changes to my repo are they automatically then included in the PR?

Reply:
I am not Bob, I am Frank.

You did it right. You pushed your 2. commit and I pulled it then. The result are 2 commits in the history.

There is also another way:
Amend the first commit and force push your branch. Then the history would show one commit.

Reply:
My apologies Frank.